src/bin/pg_autoctl/primary_standby.h

```diff
@@ -20,14 +20,18 @@
  * LocalPostgresServer represents a local postgres database cluster that
  * we can manage via a SQL connection and operations on the database
  * directory contained in the PostgresSetup.
+ *
+ * currentLSN value is kept as text for better portability. We do not
+ * perform any operation on the value after it was read from database.
+ * NAMEDATALEN is long enough for text representation of the LSN value.
```

Comment:
Maybe add some examples of normal and maximum size LSN text representation, to make it obvious that NAMEDATALEN is the proper choice here. If it is not, then maybe we can define a new size constant such as `LSN_MAXLENGTH` or something.

Reply:
pg_lsn.c defines MAXPG_LSNLEN as 17 and uses MAXPG_LSNLEN+1 length buffers to do lsn operations.

I defined PG_LSN_MAXLENGTH = 18 and used it in the code